README.md

```diff
@@ -39,17 +49,44 @@ This repository contains samples of Azure Policies that can be used as reference
   - [Remediations](https://docs.microsoft.com/rest/api/policy-insights/remediations)
   - [Guest Configuration (preview)](https://docs.microsoft.com/rest/api/guestconfiguration/)
 
-### Other
+## Other
 
 - [Video - Build 2018](https://channel9.msdn.com/events/Build/2018/THR2030)
 
-## Contributing
+## Getting Support
+
+The support role of this repository has transitioned to standard Azure support channels. See below for information about getting support help for Azure Policy.
+
+### Alias Requests
+
+An alias enables you to restrict what values or conditions are permitted for a *property* on a resource. Each alias maps to the paths in different API versions for a given resource type. During policy evaluation, the policy engine gets the property path for that API version.
+See the documentation page on aliases [**here**](https://docs.microsoft.com/en-us/azure/governance/policy/concepts/definition-structure#aliases). For additional information about Azure Policy and aliases, visit this [**blog post**](https://azure.microsoft.com/en-us/blog/more-resource-policy-aliases/)
+
+Previously, this repository was the official channel to open requests for new aliases. Since the full set of aliases for most namespaces have now been published, support for requesting aliases is now handled by Azure Customer Support. Open a new [**Azure Customer Support ticket**](https://azure.microsoft.com/en-us/support/create-ticket/) if you believe you need new aliases to be published.
+
+You can use the [```Get-AzPolicyAlias```](https://docs.microsoft.com/en-us/powershell/module/az.resources/get-azpolicyalias) Powershell cmdlet to help discover existing aliases.
+
+### General Questions
+
+If you have questions you haven't been able to answer from the [**Azure Policy documentation**](https://docs.microsoft.com/en-us/azure/governance/policy), there are a few places that host discuscussions focused on Azure Policy:
+
+ - Microsoft Azure Policy forum (something, somewhere)
```

Comment:
Looking for help about what to put here: is there any public discussion forum sponsored/hosted by Microsoft that we can send people to for community help that has Microsoft people participating?

Reply:
[Tech Community](https://techcommunity.microsoft.com) is one of the new forums, but I don't know if the team has created a 'Community' for Policy here yet (or Governance in general).

Reply:
Updated with links to the new tech community conversation space.